scalafmt-tests/src/test/resources/test/ContinuationIndentCaseSite0.case

```diff
@@ -0,0 +1,160 @@
+continuationIndent.caseSite = 0
+<<< Long if
+case tok if tok.left.name.startsWith("xml") && tok.right.name.startsWith("xml") => List( NoSplit0 )
```

Comment:
i would prefer if you put inside a `match` or a partial function, because `case` by itself is meaningless and there's likely code which checks the parent of a case statement.

Reply:
Updated, please check again.